modules/microvm/virtualization/microvm/guivm.nix

```diff
@@ -245,11 +245,8 @@ in
           if config.ghaf.guest.kernel.hardening.graphics.enable then
             pkgs.linuxPackagesFor guest_graphics_hardened_kernel
           else
-            # TODO: with pkgs.linuxPackages_latest (6.10.2) default brightness
-            # value of ghaf system will change from 96000 (max brightness) to
-            # 4800 (5% of max brightness) which need to be debugged
-            # so, currently align Kernel version which is used by ghaf-host
-            config.boot.zfs.package.latestCompatibleLinuxPackages;
+            # Linux Latest Kernel Version is 6.10.2
```

Comment:
I'm afraid flake updates will not keep the exact version in the comment up-to-date over time and maintainers likely won't remember to update it. I think remove the comment line altogether.

Reply:
Removed.